### ISSUE TITLE
Different results from publishedH2GCN  paper

### ISSUE THREAD

**Issue by gayanku** [GitEventhandler/H2GCN-PyTorch#2]
Running this code, the results are very different from H2GCN  paper (https://arxiv.org/pdf/2006.11468.pdf ) (table 5) results. Is this due to the datasets being different?
I am getting
cora      	77.3642
citeseer  	74.5585
pubmed    	87.5963
chameleon	49.4956
cornell   	73.5135
texas     	78.9189
wisconsin 	78.6275

This is the average accuracy of all 10 splits for each dataset with seed=42.

**Comment by GitEventhandler** [comment]
Hi gayanku, in my point of view, these differences may cause by two reasons:
1. This H2GCN port did not copy the hyper-parameters from original repo at the same time. Also, since the original [H2GCN](https://github.com/GemsLab/H2GCN) is using TensorFlow as its backend, which works kind of different to PyTorch. Some of the hyper-parameter may need some adjustment.
2. This repo uses datasets and splits inherited from [GeomGCN](https://github.com/graphdml-uiuc-jlu/geom-gcn), there may be some differences between the original one.
